ape-config.yaml

```diff
@@ -9,7 +9,7 @@ plugins:
 dependencies:
   - name: OpenZeppelin
     github: OpenZeppelin/openzeppelin-contracts
-    version: 4.8.1
+    version: 4.9.0
```

Comment:
is this what is used in hyperdrive?

Reply:
I think so, I saw it in my local files

Reply:
v0.0.2 of hyperdrive points to a May 19 commit, which is after the May 17 release of [v4.9.0-rc.1](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v4.9.0-rc.1)